pkg/skaffold/config/global_config.go

```diff
@@ -37,6 +37,7 @@ type ContextConfig struct {
 	KindDisableLoad      *bool         `yaml:"kind-disable-load,omitempty"`
 	K3dDisableLoad       *bool         `yaml:"k3d-disable-load,omitempty"`
 	CollectMetrics       *bool         `yaml:"collect-metrics,omitempty"`
+	UpdateCheckConfig    *UpdateConfig `yaml:"update-config,omitempty"`
```

Comment:
the `-config` part of the `update-config` yaml tag doesn't seem to mean much, maybe we could just call it `update`?

Reply:
`update` is a verb.
 Since this is a top level struct and `update-check` is already used,  i chose `update-config`. 

However, survey is also a noun and verb. I dont have a preference.

Reply:
I think it's good how it is, your explanation makes sense :)